### PR TITLE
HPCC-20019  Update Docker build for HPCC 7.0.0-beta2

### PR DESCRIPTION
**Pull request by xwang2713**
Also update Dockerfile to HPCC 6.4.22-1

Fix exec parameter in docker-enterpoint.sh. Remove bash in CMD.
Also add tag and repo optioin for test-build.sh
update README.md

@g-pan please review.

Test builds https://hub.docker.com/r/hpccsystems/hpcc-m/builds/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +59,16 @@ There are two Linux distributions: Ubuntu (Ubuntu 16.04, default) and CentOS (Ce
...
+./update 6.4.18-1
 ```
 ### Build
 You can go to the directory of interest which contains the Dockerfile to run the build, or you can run the test-build.sh script which includes test:
 ```sh
 sudo ./test-build.sh -v 6 -l ubuntu -p platform
 ```
 This will build HPCC Platform 6.x  for Ubuntu 16.04.
+
+To upload build image to Docker Hub:
```
Comment (g-pan) [inline README.md]: To upload a build image to Docker Hub:

File: README.md [hunk 2]
```diff
@@ -58,11 +59,16 @@ There are two Linux distributions: Ubuntu (Ubuntu 16.04, default) and CentOS (Ce
...
+```sh
+docker push <repo>:<tag>   For example docker push hpccsystems/hpcc:latest
```
Comment (g-pan) [inline README.md]: docker push &lt;repo&gt;:&lt;tag&gt;  For example: docker push hpccsystems/hpcc:latest

**Review by g-pan — COMMENTED** [review]
Two VERY MINOR comments. Looks good with or without suggested changes. Approved.

**Comment by richardkchapman** [comment]
@xwang2713 Please rebase for a clean merge

**Comment by xwang2713** [comment]
@richardkchapman Resolved conflicts. Ready to merge.

**Comment by richardkchapman** [comment]
@xwang2713 There's a rogue merge on this branch - looks like you merged master rather than rebasing onto it.

**Comment by xwang2713** [comment]
Let me see if I can redo it.

**Review by xwang2713 — COMMENTED** [review]
@richardkchapman I reverted the merge and rebased with master. Let me know if still have problem to merge.

**Pull request merged**